### PR TITLE
Sync translations

### PR DESCRIPTION
**Pull request by mikedld**
Add new languages (90+% complete):
* Mac client:
  * Basque (eu)
  * Hebrew (he)
  * Hungarian (hu)
  * Japanese (ja)
  * Polish (pl)
  * Portuguese (Brazil) (pt_BR)
  * Swedish (sv)
  * Ukrainian (uk)
  * Chinese (China) (zh_CN)
  * Chinese (Taiwan) (zh_TW)
* Qt client:
  * Icelandic (is)

Fix Qt client languages list to include those already in translations/ subdirectory and 90+% complete:
* Hebrew (he)
* Chinese (Taiwan) (zh_TW)

Remove Italian (Italy) (it_IT) from Qt client, add Italian (it) instead.

Fixes: #5189
Fixes: #5759
Fixes: #6284

### CODE REVIEW

**Comment by tearfur** [comment]
fyi Mike, you have to put "fixes" in front of every issue to link them all to this PR.

**Comment by mikedld** [comment]
Yeah, I noticed, thanks :-\

File: Transmission.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2786,6 +2916,16 @@
 				"pt-PT",
 				pt_PT,
```
Comment (Coeur) [inline Transmission.xcodeproj/project.pbxproj:2917]: Let's merge them by only keeping one of the two?

File: macosx/fr.lproj/Localizable.strings [hunk 2]
```diff
@@ -685,6 +693,9 @@
 /* Preferences -> Network -> port status */
 "Port is open" = "Le port est ouvert";
 
+/* Inspector -> Peers tab -> table row tooltip */
+/* "Port: %@" = ""; */
```
Comment (Coeur) [inline macosx/fr.lproj/Localizable.strings:697]: The translation was present previously (line 676-677), so why causing a regression by removing it?

**Review by Coeur — APPROVED** [review]

**Comment by mikedld** [comment]
Decided to go with hyphens as separators for Mac client (although doesn't seem to be strictly required and works either way) after reading https://developer.apple.com/documentation/xcode/choosing-localization-regions-and-scripts:

> To distinguish between different languages and regional variants, use a language code with a region code separated by hyphens (`[language code]-[region code]`).

File: .tx/config [hunk 3]
```diff
@@ -12,72 +12,84 @@ file_filter = macosx/<lang>.lproj/Localizable.strings
 source_file = macosx/en.lproj/Localizable.strings
 source_lang = en
 type        = STRINGS
+lang_map    = pt_BR: pt-br, pt_PT: pt-PT, zh_CN: zh-CN, zh_TW: zh-TW
```
Comment (nevack) [inline .tx/config:15]: btw, why `br` in `pt-br` is lowercase?

Comment (mikedld) [reply .tx/config]: Because I need to sleep more :) Thanks!

**Review by nevack — APPROVED** [review]
LGTM

**Pull request merged**